protobuf/README.md

```diff
@@ -0,0 +1,22 @@
+# @hwsc/hwsc-api-blocks
+An npm package that contains proto client stubs for all services for `hwsc-frontend`
+
+## Installation
+1. `$ npm i @hwsc/hwsc-api-blocks-lisa-test`
```

Comment:
Make sure you change name of the npm package and any associated login username later on.